spec/acceptance/ace/transport_app_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'ace/transport_app'
+require 'rack/test'
+require 'ace/config'
+
+RSpec.describe ACE::TransportApp do
+  include Rack::Test::Methods
+
+  def app
+    ACE::TransportApp.new(ACE::Config.new(base_config))
+  end
+
+  let(:base_config) do
+    {
+      "ssl-cert" => "spec/volumes/puppet/ssl/certs/aceserver.pem",
+      "ssl-key" => "spec/volumes/puppet/ssl/private_keys/aceserver.pem",
+      "ssl-ca-cert" => "spec/volumes/puppet/ssl/certs/ca.pem",
+      "ssl-ca-crls" => "spec/volumes/puppet/ssl/ca/ca_crl.pem",
+      "puppet-server-uri" => "https://localhost:8140",
+      "cache-dir" => "tmp/"
+    }
+  end
+
+  let(:execute_catalog_body) do
+    {
+      "target": {
+        "remote-transport": "panos",
+        host: certname,
+        user: "admin",
+        password: "admin",
+        ssl: false
+      },
+      "compiler": {
+        "certname": certname,
+        "environment": "production",
+        "transaction_uuid": Random.new_seed.to_s,
```

Comment:
manual verification finds that this is throwing an error on the reporting side:

`2019-04-12 14:47:28,622 ERROR [p.p.command] [5] [store report] Retrying after attempt 0 for livupbi6o63eyg3.delivery.puppetlabs.net, due to: org.postgresql.util.PSQLException: ERROR: invalid input syntax for uuid: "255097297177477792347640402086542686387"`

The suggestion is to replace with a hard-coded UUID value e.g. "2d931510-d99f-494a-8c67-87feb05e1594"

Reply:
Thanks for finding it, not sure why this wasn't shown, was this trailing the PuppetDB logs? The acceptance test is going be thrown away afterwards, was just to have it contained somewhere to verify the refactor etc.

Reply:
Ah yes, I see it now.